src/neo4j/_api.py

```diff
@@ -130,21 +130,18 @@ class NotificationDisabledCategory(str, Enum):
     Inherits from :class:`str` and :class:`Enum`. Every driver API accepting a
     :class:`.NotificationFilter` value will also accept a string::
 
-        >>> NotificationDisabledCategory.HINT == "HINT"
-        True
         >>> NotificationDisabledCategory.UNRECOGNIZED == "UNRECOGNIZED"
         True
-        >>> NotificationDisabledCategory.UNSUPPORTED == "UNSUPPORTED"
-        True
         >>> NotificationDisabledCategory.PERFORMANCE == "PERFORMANCE"
         True
         >>> NotificationDisabledCategory.DEPRECATION == "DEPRECATION"
         True
-        >>> NotificationDisabledCategory.GENERIC == "GENERIC"
-        True
 
     .. versionadded:: 5.7
 
+    .. versionchanged:: 5.14
+        Added categories :attr:`.SECURITY` and :attr:`.TOPOLOGY`.
```

Comment:
Is it worth adding that it shouldn't be used with server versions before 5.14?

Reply:
Good point. Added.